### PR TITLE
docs: Update link to DROP KEYSPACE instead of DROP TYPE within the YCQL drop_type page

### PR DESCRIPTION
**Pull request by dr0pdb**

### CODE REVIEW

**Review by ddorian — APPROVED** [review]
LGTM

File: docs/content/preview/api/ycql/ddl_drop_type.md [hunk 1]
```diff
@@ -51,4 +51,3 @@ ycqlsh:example> DROP TYPE person;
 ## See also
 
 - [`CREATE TABLE`](../ddl_create_table)
-- [`DROP TYPE`](../ddl_drop_keyspace)
```
Comment (aishwarya24) [inline docs/content/preview/api/ycql/ddl_drop_type.md:54]: The link is for DROP KEYSPACE. You need to change the title to [`DROP KEYSPACE`](../ddl_drop_keyspace) everywhere

Comment (dr0pdb) [reply docs/content/preview/api/ycql/ddl_drop_type.md:54]: Done.

**Review by ddhodge — APPROVED** [review]
LGTM

**Pull request merged**